api/src/main/scala/com/rasterfoundry/granary/Server.scala

```diff
@@ -57,6 +58,13 @@ object ApiServer extends IOApp {
           case Right(config) => IO.pure(config)
         }
       }
+      authConfig <- Resource.liftF {
+        ConfigSource.default.at("auth").load[AuthConfig] match {
+          case Left(e) =>
+            IO.raiseError(new Exception(e.toList.map(_.toString).mkString("\n")))
```

Comment:
I think probably I'm responsible for this pattern, but there's a better way maybe --

We can `leftMap` the `Either` to an `Either[Throable, AuthConfig]`, and then call `IO.fromEither` on the result -- https://typelevel.org/cats-effect/datatypes/io.html#fromeither

I have some regrets

Reply:
In the interest of getting this merged soon, I think I'm gonna punt on this.